mmrotate/evaluation/metrics/dota_metric.py

```diff
@@ -163,10 +171,12 @@ def merge_results(self, results: Sequence[dict],
             for f, dets in zip(file_objs, dets_per_cls):
                 if dets.size == 0:
                     continue
-                bboxes = obb2poly_np(dets, self.version)
-                for bbox in bboxes:
-                    txt_element = [img_id, str(bbox[-1])
-                                   ] + [f'{p:.2f}' for p in bbox[:-1]]
+                th_dets = torch.from_numpy(dets)
+                rboxes, scores = torch.split(th_dets, (5, 1), dim=-1)
+                qboxes = RotatedBoxes(rboxes).convert_to('qbox').tensor
```

Comment:
Can directly use rbox2qbox in box_converters.py

Reply:
fix in #472